en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup-with-km-separated.md

```diff
@@ -0,0 +1,912 @@
+# Configuring a Distributed API-M Deployment with Key Manager and Traffic Manager Separation
+
+This document describes how to configure a distributed WSO2 API Manager deployment with the Key Manager and Traffic Manager separated from the Control Plane for migrated environments using Opaque tokens.
+
+The following diagram shows the communications between the Control Plane, Gateway, Key Manager and the Traffic Manager nodes in the distributed deployment of API Manager for a migrated environment using Opaque (Reference) tokens.
```

Comment:
I think we should modify this to have a not saying separate km is needed only if you're migrating from an old version AND using opaque tokens. @tharikaGitHub WDYT?

Reply:
@samgnaniah 

1. Let’s add the note after step 6 in https://apim.docs.wso2.com/en/latest/install-and-setup/upgrading-wso2-api-manager/upgrading-process/

2. Bold the following paragraph in the note.

When migrating from previous versions of WSO2 API Manager, if you are still willing to continue the use of Opaque (Reference) tokens, you will need to maintain a Gateway to Key Manager mapping where when the Gateways are scaled, the Key Manager should also be scaled.

3. As the heading of the note, let’s add / highlight
 
The following is only applicable if you are migrating from an older version and at the same time wants to continue using Opaque(Reference) tokens in your distributed deployment.

Reply:
My comment was not about the "upgrading-process" page though. It was about the new page.